app/views/content_item_signups/confirm.html.erb

```diff
@@ -31,9 +31,18 @@
       You can choose how often you want to receive emails.
     </p>
 
-    <%= form_tag(:action => :create) do %>
+    <%= form_tag(action: :create) do %>
       <%= hidden_field_tag 'link', @content_item['base_path'] %>
-      <%= submit_tag('Sign up', class: 'govuk-button') %>
+      <%= render 'govuk_publishing_components/components/button', {
+        text: 'Sign up',
+        margin_bottom: true,
+        data_attributes: {
+          module: 'auto-track-event',
+          track_category: 'email_subscriptions',
```

Comment:
For these tracking attributes to work, you can just use module: "track-click" instead of "auto-track-event" as we're doing standard tracking here (nothing custom).